iroh/src/magicsock.rs

```diff
@@ -518,8 +518,11 @@ impl MagicSock {
                             udp_sent = true;
                         }
                         Err(err) => {
-                            error!(node = %node_id.fmt_short(), dst = %addr,
-                                   "failed to send udp: {err:#}");
+                            // No need to print "WouldBlock" errors to the console
+                            if err.kind() != io::ErrorKind::WouldBlock {
+                                error!(node = %node_id.fmt_short(), dst = %addr,
```

Comment:
I'd even turn this down to WARN.

Reply:
Hm, I don't mind seeing error logs for non-fatal errors from the UDP socket.

Reply:
I think ERROR is something that you essentially can consider the `Endpoint` to become unusable.  It is reasonable to expect now that you should destroy it and create a new one.  This is clearly not an error because the socket could recover and continue.